processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -651,6 +652,12 @@ public Object get()
     }
 
     @Override
+    public NumericColumnSelector makeMathExpressionSelector(String expression)
+    {
+      throw new UnsupportedOperationException("expression is not supported in current context");
```

Comment:
IIRC this is needed to make expressions work on the outer query of a nested groupBy. could you double-check that works by adding something to GroupByQueryRunnerTest?